File: dotnet/az-sql-server/az-sql-server-lib/AZSQLDbContext.cs
using Microsoft.EntityFrameworkCore;
using System;
using System.Collections.Generic;

namespace az_sql_server
{
    internal class AZSQLDbContext : DbContext
    {
        private readonly string dbConnString;

        public AZSQLDbContext(string dbConnString) {
            this.dbConnString = dbConnString;
            this.Database.EnsureCreated();
        }

        internal DbSet<TaskExecutionInformation> TaskExecutionInformation { get; set; }

        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
        {            
            optionsBuilder.UseSqlServer(dbConnString);                
        }
    }
}

File: java/az-sql-server-dal/src/main/java/dk/commentor/dal/azsql/AZSQLDbContext.java
package dk.commentor.dal.azsql;

class AZSQLDbContext implements DbContext {
    private final String dbConnString;
    DbSet<OrderDTO>Orders;

    public AZSQLDbContext(String dbConnString) {
        this.dbConnString = dbConnString;
        this.Database.EnsureCreated();
    }

    protected void OnConfiguring(DbContextOptionsBuilder optionsBuilder) {
        optionsBuilder.UseSqlServer(dbConnString);
    }
}
